tests/testing_harness.py

```diff
@@ -0,0 +1,245 @@
+from __future__ import print_function
+
+import filecmp
+import glob
+import hashlib
+from optparse import OptionParser
+import os
+from subprocess import Popen, STDOUT, PIPE, call
+import sys
+
+import numpy as np
+
+sys.path.insert(0, '../..')
```

Comment:
When a test_*.py file imports `TestHarness`, it's cwd is the test_* directory.  So we need both '..'s.